doctrine/events.rst

```diff
@@ -0,0 +1,445 @@
+                        * 'entity_manager': define it if the listener is not associated to the
+                        *                   default entity manager
+                        * 'method': by default, Symfony looks for a method called after the event (e.g. postUpdate())
+                        *           if it doesn't exist, it tries to execute the '__invoke()' method, but
+                        *           you can configure a custom method name with the 'method' option
+                    -->
+                    <tag name="doctrine.orm.entity_listener"
+                        event="postUpdate"
+                        entity="App\Entity\User"
+                        lazy="true"
+                        entity_manager="custom"
+                        method="checkUserChanges"/>
+                </service>
+            </services>
+        </container>
+
+    .. code-block:: php
+
+        use App\Entity\User;
+        use App\EventListener\UserChangedNotifier;
+
+        $container->autowire(UserChangedNotifier::class)
+            ->addTag('doctrine.orm.event_listener', [
+                // these are the basic options that define the entity listener
+                'event' => 'postUpdate',
+                'entity' => User::class,
+
+                // set the 'lazy' option to TRUE to only instantiate listeners when they are used
+                'lazy' => true,
+
+                // you can also associate an entity listener to a specific entity manager
+                'entity_manager' => 'custom',
+
+                // by default, Symfony looks for a method called after the event (e.g. postUpdate())
+                // if it doesn't exist, it tries to execute the '__invoke()' method, but you can
+                // configure a custom method name with the 'method' option
+                'method' => 'checkUserChanges',
+            ])
+        ;
+
+Doctrine Lifecycle Subscribers
+------------------------------
+
+Lifecycle subscribers are defined as PHP classes that implement the
+``Doctrine\Common\EventSubscriber`` interface and which listen to one or more
+Doctrine events on all the application entities. For example, suppose that you
+want to log all the database activity. To do so, define a listener for the
+``postPersist``, ``postRemove`` and ``postUpdate`` Doctrine events::
+
+    // src/EventListener/DatabaseActivitySubscriber.php
+    namespace App\EventListener;
+
+    use App\Entity\Product;
+    use Doctrine\Common\EventSubscriber;
+    use Doctrine\Common\Persistence\Event\LifecycleEventArgs;
+    use Doctrine\ORM\Events;
+
+    class DatabaseActivitySubscriber implements EventSubscriber
+    {
+        // this method can only return the event names; you cannot define a
+        // custom method name to execute when each event triggers
+        public function getSubscribedEvents()
+        {
+            return [
+                Events::postPersist,
+                Events::postRemove,
```

Comment:
what about this order: persist then update then remove?
if change, also change on the methods order

Reply:
I added in alphabetical order ... which is what we usually do in Symfony Docs in cases like this.